### PR TITLE
Creating stable-release_iot Folder for celadon_IoT external releases

### PR DESCRIPTION
**Pull request by usivarax**
creation of Celadon IoT release details documentation for external
releases , Initial push for documentation
Below Pages to be created : 
Stable Release (IoT)
         Feature Delta Offered by Celadon IoT
         Use Cases & Market Segments
         Release Notes

Signed-off-by: Sivaramakrishnan, UdhayanX <udhayanx.sivaramakrishnan@intel.com>

### CODE REVIEW

**Comment by swaroopbalan** [comment]
•	https://github.com/usivarax/celadon-documentation/blob/celadon_IoT-initial/source/stable-release_iot/stable-release_iot.rst  there are some extra lines at the below 
o	.. toctree::
o	:maxdepth: 2

o	feature-delta-iot
o	usecases-market-segment
o	release-notes
•	https://github.com/usivarax/celadon-documentation/blob/celadon_IoT-initial/source/stable-release_iot/stable-release_iot.rst 
o	here I think Yogesh told not to use CML TGL etc
•	https://github.com/usivarax/celadon-documentation/blob/celadon_IoT-initial/source/stable-release_iot/release-notes.rst 
o	here there are indentation issue 
o	Supplementary guide: Power and volume key support
•	https://github.com/usivarax/celadon-documentation/blob/celadon_IoT-initial/source/stable-release_iot/feature-delta-iot.rst 
o	lot of indentation issue

File: source/stable-release_iot/feature-delta-iot.rst [hunk 1]
```diff
@@ -0,0 +1,76 @@
+.. _feature-delta-iot:
+
+Feature Delta Offered by Celadon IoT
```
Comment (yogesh-marathe) [inline source/stable-release_iot/feature-delta-iot.rst:3]: Hi @usivarax, I still see we are prominently using 'Celadon IoT' every where. There has been alignment on using 'Stable Releases (IoT), Would you please search replace 'Celadon IoT' with 'Stable Releases (IoT)'

File: source/stable-release_iot/stable-release_iot.rst [hunk 2]
```diff
@@ -0,0 +1,17 @@
+.. _stable-celadon-iot:
+
+Stable Celadon IoT
+##################
+
+Celadon IoT is an Internet of Things (IoT) overlay on top of Celadon. Celadon IoT, release provides users with an extensive set of features on Comet Lake(CML), Tiger Lake(TGL), and Elkhart Lake(EHL) Intel® platforms with single Baseline/Image.
+
+Celadon IoT offers 100% compliance for Compatibility Test Suite (CTS), Vendor Test Suite (VTS), CTS-on-GSI, GMS Test Suite (GTS), and Security Test Suite (STS) on Comet Lake(CML), Tiger Lake(TGL), and Elkhart Lake(EHL).
```
Comment (yogesh-marathe) [inline source/stable-release_iot/stable-release_iot.rst:8]: This is repetitive, either cover here or in feature delta iot that you have already covered.

File: source/stable-release_iot/stable-release_iot.rst [hunk 3]
```diff
@@ -0,0 +1,17 @@
+.. _stable-celadon-iot:
+
+Stable Celadon IoT
+##################
+
+Celadon IoT is an Internet of Things (IoT) overlay on top of Celadon. Celadon IoT, release provides users with an extensive set of features on Comet Lake(CML), Tiger Lake(TGL), and Elkhart Lake(EHL) Intel® platforms with single Baseline/Image.
+
+Celadon IoT offers 100% compliance for Compatibility Test Suite (CTS), Vendor Test Suite (VTS), CTS-on-GSI, GMS Test Suite (GTS), and Security Test Suite (STS) on Comet Lake(CML), Tiger Lake(TGL), and Elkhart Lake(EHL). 
+
+Celadon IoT empowers the community to utilize 100%compliance on the supported platforms with continuous support for new features and fixes.
```
Comment (yogesh-marathe) [inline source/stable-release_iot/stable-release_iot.rst:10]: "continuous support for new features and fixes" -> I would just say "continuous updated with ASBs", are you intending to develop new features on this Stable Releases(IoT) branch while in maintenance phase?

File: source/stable-release_iot/feature-delta-iot.rst [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+####################################
+
+Following are the additional features provided by Celadon IoT.
+
+.. contents::
+    :depth: 1
+    :local:
+* 100% Google Conformance on Comet Lake(CML),Tiger Lake(TGL) & Elkhart Lake(EHL) with Production ready
```
Comment (yogesh-marathe) [inline source/stable-release_iot/feature-delta-iot.rst:11]: Production ready - there was an action to confirm if you can claim this with SDL completion. Is that done now?

File: source/stable-release_iot/feature-delta-iot.rst [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+* Supporting Comet Lake(CML),Tiger Lake(TGL) & Elkhart Lake(EHL) on single baseline and Single Image
+* Elkhart Lake(EHL) Platform supported on Celadon with 100% compliance
```
Comment (yogesh-marathe) [inline source/stable-release_iot/feature-delta-iot.rst:13]: Why are we calling out EHL again, we covered 100% conformance and EHL in above two points already. Do we have a conclusion on the public platform that can be used along with this release?

File: source/stable-release_iot/feature-delta-iot.rst [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+* IoT configuration support GVT-d mode with 100% conformance
+* IoT configuration support GVT-g with no conformance
+
+Real-time Clock (RTC) & Alarm Virtualization
+********************************************
+
+Celadon in Virtual Machine (CiV) project may requires operations on the RTC device, such as changes to current time and setups of alarm, to be synced to host and persistent across reboots. RTC device is purely emulated by QEMU and its state will be lost after exit of QEMU process, furthermore, guest RTC alarm is unable to wake up the guest when host is also suspended, which is the case in when using --host-pm-control option in CiV.
```
Comment (yogesh-marathe) [inline source/stable-release_iot/feature-delta-iot.rst:20]: may require

"which is the case in when using --host-pm-control option in CiV" please check for readability

File: source/stable-release_iot/feature-delta-iot.rst [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+
+As such, due to emulation, secure partition erase would be fall back to “byte-by-byte overwrite” to the virtual partition. This “byte-by-byte overwrite” operation is not solid state storage friendly.
+
+To provide secure data erase of data partition, following enhancement is needed in CiV:
+
+Addition of secure data erase to make use of storage controller hardware secure erase feature, when it is supported in hardware to erase data partition and only finally fall back to byte-by-byte override 0 when not supported during Android recovery/wipe data process.
+
+#. Guest virtual disk image file enhancement
+
+   In Celadon CiV, all Android required partitions are resided in the same guest virtual disk image file in host.
```
Comment (yogesh-marathe) [inline source/stable-release_iot/feature-delta-iot.rst:53]: what is Celadon CiV? CiV is Celadon in VM.

**Comment by swaroopbalan** [comment]
hi Venu 
We were under the assumption that release notes will be added here 
https://github.com/usivarax/celadon-documentation/blob/celadon_IoT-initial/source/release-notes.rst 


but seems like you added separatelyhttps://github.com/usivarax/celadon-documentation/blob/celadon_IoT-initial/source/stable-release_iot/release-notes.rst

please list high level Celadon release notes to have Stable, Base, Deprecated
K, Venugopal Under Stable Releases (IoT) list your releases

File: source/stable-release_iot/feature-delta-iot.rst (hunk 1 above)
Comment (usivarax) [reply source/stable-release_iot/feature-delta-iot.rst]: Updated as per the Suggestion

File: source/stable-release_iot/stable-release_iot.rst (hunk 2 above)
Comment (usivarax) [reply source/stable-release_iot/stable-release_iot.rst]: Updated as per the Suggestion: Removed 2nd Bulletin from stable-release_iot.rst file

File: source/stable-release_iot/stable-release_iot.rst (hunk 3 above)
Comment (usivarax) [reply source/stable-release_iot/stable-release_iot.rst]: Updated as per the Suggestion

File: source/stable-release_iot/feature-delta-iot.rst (hunk 4 above)
Comment (usivarax) [reply source/stable-release_iot/feature-delta-iot.rst:11]: SDL Completion is Done

File: source/stable-release_iot/feature-delta-iot.rst (hunk 5 above)
Comment (usivarax) [reply source/stable-release_iot/feature-delta-iot.rst]: Updated as per the Suggestion : Removed the Bulletin

File: source/stable-release_iot/feature-delta-iot.rst (hunk 7 above)
Comment (usivarax) [reply source/stable-release_iot/feature-delta-iot.rst]: Updated as per the Suggestion

File: source/stable-release_iot/feature-delta-iot.rst (hunk 6 above)
Comment (yogesh-marathe) [reply source/stable-release_iot/feature-delta-iot.rst:19]: @usivarax Yet to be fixed?

**Comment by yogesh-marathe** [comment]
@swaroopbalan few minor comments still pending (I have added them), should be good to go after those comments are addressed.

File: source/stable-release_iot/feature-delta-iot.rst (hunk 6 above)
Comment (usivarax) [reply source/stable-release_iot/feature-delta-iot.rst]: As suggested by @sfonn modified the line before "--host-pm-control": 
RTC device is purely emulated by QEMU and its state will be lost after exit of QEMU process. Furthermore, guest RTC alarm is unable to wake up guest after host suspends.

**Comment by yogesh-marathe** [comment]
LGTM, good to merge

**Review by swaroopbalan — APPROVED** [review]

**Pull request merged**